src/Frontend/Page/Idea.hs

```diff
@@ -169,6 +169,13 @@ linkToIdeaLocation idea = do
              IdeaLocationSpace{} -> "Zum Ideenraum"
              IdeaLocationTopic{} -> "Zum Thema"
 
+-- ASSUMPTION: The caps list does con contain the same value marked as
+-- Clickable and GrayedOut
```

Comment:
i like, in this order:

1. eitherClickable
1. eitherClickableGrayedOut
1. clickable

all are ok for me.